.gitignore

```diff
@@ -8,4 +8,5 @@ yarn-error.log
 .DS_Store
 /cypress/screenshots/
 /cypress/videos/
-target
\ No newline at end of file
+target
+.eslintcache
```

Comment:
You might need to remove this once the overview page is merged, otherwise there will be conflicts. The same applies for opensearch_dashboards.json and package.json.